sklearn/discriminant_analysis.py

```diff
@@ -649,12 +651,10 @@ def fit(self, X, y, store_covariances=None, tol=None):
             Target values (integers)
         """
         if store_covariances:
-            warnings.warn("The parameter 'store_covariances' is deprecated as "
-                          "of version 0.17 and will be removed in 0.19. The "
-                          "parameter is no longer necessary because the value "
-                          "is set via the estimator initialisation or "
-                          "set_params method.", DeprecationWarning)
-            self.store_covariances = store_covariances
+            warnings.warn("'store_covariances' was renamed to 'store_covariance'"
```

Comment:
You need to update your copy of master and merge with it. You'll find this code path has disappeared.